_rules/aria-required-context-role-ff89c9.md

```diff
@@ -206,16 +206,11 @@ There is no element with an [explicit role][].
 
 #### Inapplicable Example 3
 
-This `section` element with an [explicit role][] of `doc-biblioentry` has a role from the [Digital Publishing WAI-ARIA Module (DPUB ARIA) 1.0][dpub 1.0], not the [WAI-ARIA 1.1 Recommendation][aria 1.1].
+This `section` element with an [explicit role][] of `doc-abstract` has a role from the [Digital Publishing WAI-ARIA Module (DPUB ARIA) 1.0][dpub 1.0], not the [WAI-ARIA 1.1 Recommendation][aria 1.1].
 
 ```html
-<section role="doc-bibliography">
-	<h1>Cited Works</h1>
-	<div role="list">
-		<p role="doc-biblioentry" id="b8cab5dd-bc24-459c-9858-7afa9da69b64">
-			John Steinbeck, The Grapes of Wrath (New York: The Viking Press, 1939)
-		</p>
-	</div>
+<section role="doc-abstract" aria-label="Abstract">
+	<p>Accessibility of web content requires semantic information about widgets, structures, and behaviors …</p>
```

Comment:
I disagree with changing this. The point of this example is to show a DPUB role that has a required context that is inapplicable.

Reply:
But in that example, the `<div role="list">` owns no element with a role of `listitem`. Because role inherintance doesn't work like that and ARIA DPUB 1.0 is broken on that point, and this is known, and this is the reason we advance for excluding DPUB roles. See also https://github.com/w3c/dpub-aria/issues/15 for more details.

So, we have an Inapplicable example that does not satisfy the Accessibility Requirement (1.3.1), which goes directly against our test cases design guidance… https://github.com/act-rules/act-rules.github.io/blob/develop/pages/design/test-cases.md
> This requires that **passed and inapplicable test cases should satisfy the accessibility requirements** of the rule

(emphasise is not mine)

So that example should be killed anyway.

Reply:
Aaahhh. I see what you mean. Sorry, took me a while. Yeah that's a fair point. That `list` is invalid here.